iscsi/targetd/README.md

```diff
@@ -231,11 +231,10 @@ Run the following commands. The secret correspond to username and password you h
 ```
 oc new-project iscsi-provisioner
 oc create sa iscsi-provisioner
-# if Openshift is version < 3.6 add the iscsi-provisioner-runner role
```

Comment:
ditto, this change looks unintended as well

Reply:
done